NOTE: this is an hack not an fix.. we do not use LVM* so should not matter for now
see: https://github.com/calamares/calamares/issues/1061

diff -Naur 3.2.2-187/src/modules/partition/gui/PartitionPage.cpp 3.2.2-187-p/src/modules/partition/gui/PartitionPage.cpp
--- 3.2.2-187/src/modules/partition/gui/PartitionPage.cpp	2019-01-02 17:45:42.000000000 +0100
+++ 3.2.2-187-p/src/modules/partition/gui/PartitionPage.cpp	2019-01-02 21:19:58.222228061 +0100
@@ -164,35 +164,7 @@
 
     if ( m_ui->deviceComboBox->currentIndex() >= 0 )
     {
-        QModelIndex deviceIndex = m_core->deviceModel()->index( m_ui->deviceComboBox->currentIndex(), 0 );
-        auto device = m_core->deviceModel()->deviceForIndex( deviceIndex );
-        if ( device->type() != Device::Type::LVM_Device )
-        {
-            createTable = true;
-
-#ifdef WITH_KPMCOREGT33
-            if ( device->type() == Device::Type::SoftwareRAID_Device &&
-                 static_cast< SoftwareRAID* >(device)->status() == SoftwareRAID::Status::Inactive )
-            {
-                createTable = false;
-                create = false;
-            }
-#endif
-        }
-        else
-        {
-            currentDeviceIsVG = true;
-
-            LvmDevice* lvmDevice = dynamic_cast<LvmDevice*>(m_core->deviceModel()->deviceForIndex( deviceIndex ));
-
-            isDeactivable = DeactivateVolumeGroupOperation::isDeactivatable( lvmDevice );
-            isRemovable = RemoveVolumeGroupOperation::isRemovable( lvmDevice );
-
-            isVGdeactivated = m_core->isVGdeactivated( lvmDevice );
-
-            if ( isVGdeactivated )
-                m_ui->revertButton->setEnabled( true );
-        }
+        createTable = true;
     }
 
     m_ui->createButton->setEnabled( create );
